xrpl/core/binarycodec/types/issue.py

```diff
@@ -52,6 +54,13 @@ def from_value(cls: Type[Self], value: Dict[str, str]) -> Self:
             issuer_bytes = bytes(AccountID.from_value(value["issuer"]))
             return cls(currency_bytes + issuer_bytes)
 
+        if MPTAmount.is_dict_of_model(value):
+            mpt_issuance_id_bytes = bytearray()
+            Hash192.from_value(value["mpt_issuance_id"]).to_byte_sink(
+                mpt_issuance_id_bytes
+            )
```

Comment:
```suggestion
            mpt_issuance_id_bytes = bytes(Hash192.from_value(value["mpt_issuance_id"]))
```

Clarification: Is there any difference between these two snippets? All the unit tests pass.

I'm only asking to better understand the code. I don'a have a preference for either.